packages/turf-area/test.ts

```diff
@@ -21,6 +21,37 @@ const fixtures = fs.readdirSync(directories.in).map((filename) => {
   };
 });
 
+// fixtures
+
+const invalidPoly = geometry("Polygon", [
+  [
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+  ],
+]);
+
+// compiler does not pass coords less than 4 point long, so have to re-assign
+invalidPoly.coordinates = [
+  [
+    [101.0, 0.0],
+    [101.0, 0.5],
+    [101.5, 0.5],
+  ],
+];
+
+const rotatingPoly = polygon([
+  [
+    [28.321755510202507, 16.35627490376781],
+    [20.424575867090823, 1.7575215418945476],
+    [48.254218513706036, 20.42650462625916],
+    [36.310934132380964, 14.226760576846956],
+    [28.321755510202507, 16.35627490376781],
+  ],
+]);
```

Comment:
Not required, but if these test inputs are only used in a single test, consider moving the definitions down into the test they are used, so that the tests are more self-contained.  I think it makes the tests easier to read.